v22.2/import-into.md

```diff
@@ -5,25 +5,75 @@ toc: true
-- `IMPORT INTO` does not offer `SELECT` or `WHERE` clauses to specify subsets of rows. To do this, use [`INSERT`](insert.html#insert-from-a-select-statement).
+- `IMPORT INTO` invalidates all [foreign keys](foreign-key.html) on the target table. To validate the foreign key(s), use the [`ALTER TABLE ... VALIDATE CONSTRAINT`](alter-table.html#validate-constraint) statement.
+- `IMPORT INTO` is an insert-only statement; it cannot be used to [update](update.html) existing rows. Imported rows cannot conflict with primary keys in the existing table, or any other [`UNIQUE`](unique.html) constraint on the table.
+- `IMPORT INTO` does not offer `SELECT` or `WHERE` clauses to specify subsets of rows. To add a subset of rows to a table, use [`INSERT`](insert.html#insert-from-a-select-statement).
 - `IMPORT INTO` will cause any [changefeeds](change-data-capture-overview.html) running on the targeted table to fail.
 - `IMPORT INTO` supports importing into [`REGIONAL BY ROW`](alter-table.html#regional-by-row) tables.
-- See the [`IMPORT`](import.html) page for guidance on importing PostgreSQL and MySQL dump files.
 
-{{site.data.alerts.callout_info}}
+{{site.data.alerts.callout_success}}
 Optimize import operations in your applications by following our [Import Performance Best Practices](import-performance-best-practices.html).
 {{site.data.alerts.end}}
 
+## Before you begin
+
+Before using `IMPORT INTO`, you should have:
+
+- An existing CockroachDB table to import into. `IMPORT INTO` supports [computed columns](computed-columns.html) and certain [`DEFAULT` expressions](#supported-default-expressions).
+
+- Sufficient capacity in the [CockroachDB store](#available-storage) for the imported data.
+
+- The CSV or Avro data you want to import, preferably hosted on cloud storage. The [import file location](#import-file-location) must be equally accessible to all nodes using the same import file location. This is necessary because the `IMPORT INTO` statement is issued once by the client, but is executed concurrently across all nodes of the cluster.
+
+### Supported `DEFAULT` expressions
+
+`IMPORT INTO` supports the following [`DEFAULT`](default-value.html) expressions:
+
+- `DEFAULT` expressions with [user-defined types](enum.html).
+
+- Constant `DEFAULT` expressions, which are expressions that return the same value in different statements. Examples include:
+
+    - Literals (booleans, strings, integers, decimals, dates)
+    - Functions where each argument is a constant expression and the functions themselves depend solely on their arguments (e.g., arithmetic operations, boolean logical operations, string operations).
+
+- Current [`TIMESTAMP`](timestamp.html) functions that record the transaction timestamp, which include:
+
+    - `current_date()`
+    - `current_timestamp()`
+    - `localtimestamp()`
+    - `now()`
+    - `statement_timestamp()`
+    - `timeofday()`
+    - `transaction_timestamp()`
+
+- `random()`
```

Comment:
It looks like functions from `random()` onwards are their own things following from "supports the following default expressions:".... 

If that's the case, I think it's a little tricky to read a sub-list of functions and then function bullets that are bumped out a tab, without kind of confusing them as you read. Maybe worth switching the order round to avoid that?